canonical/store-components
# Add top borders for PackageCards

## Done
Added a top border for bundles and highlighted package cards

## QA
- Go to https://store-components-8.demos.haus/?path=/story/packagecard--highlighted and https://store-components-8.demos.haus/?path=/story/packagecard--bundle
- Check that there is an example of a highlighted card and a bundle, both with top coloured borders

## Issue
Fixes https://warthogs.atlassian.net/browse/WD-1624
